cogs/Bot.py

```diff
@@ -237,15 +237,15 @@ async def promise(self,ctx):
 
   @commands.command(brief="Privacy Policy", aliases = ["privacy"])
   async def promises(self, ctx):
-    embed = discord.Embed(title = "Privacy Policies", color = random.randint(0, 16777215))
-    embed.add_field(name = "1:", value = "We have a channel that logs when the bot joins or leaves a guild", inline = True)
-    embed.add_field(name = "2:", value = "We will store any errors(which may go back to you, but this is console only.) This is in memory anyway, thus it doesn't store in any of our DBs.")
-    embed.add_field(name = "3:", value = "we will only store user ids(in the future for balance commands and economy commands(opt in only)", inline = True)
-    embed.add_field(name = "4:", value = "We will store user id, and channel id(our channel), and last active, for ticket support if you need support. (is opt in)", inline = True)
-    embed.add_field(name = "5:", value = "we Finally store invalid commands", inline = True)
-    embed.add_field(name = "6:", value = "We may temporarily look at your mutual guilds with our bot or the list of servers our bot is in, with some information, just to test some things(like if a command is going hayware) or prevent abuse. If you want to look at the embeds with this, just ask, We will show you.", inline = True)
+    embed = discord.Embed(title = "Privacy Policy", color = random.randint(0, 16777215))
+    embed.add_field(name = "1:", value = "We have a logging channel that notifies us when the bot joins or leaves a guild", inline = True)
+    embed.add_field(name = "2:", value = "We log errors that occured in the bot(although it might contain private information). This is only visible temporarily in console, and is not stored in any of our DBs.")
+    embed.add_field(name = "3:", value = "We store user ids for economy commands (Opt In)", inline = True)
+    embed.add_field(name = "4:", value = "We store user id , and support channel id, as well as last time it was used(for archive reasons), for allowing ticket based support. (Opt In)", inline = True)
+    embed.add_field(name = "5:", value = "We store inputted invalid commands", inline = True)
+    embed.add_field(name = "6:", value = "We may temporarily look at your mutual guilds with our bot or the list of servers our bot is in, with some information, just to check if any problems occure(like if a command is going hayware) or to prevent abuse. If you want to look at what is sent in embeds, just ask, We will show you.", inline = True)
```

Comment:
error here with occured, it should be occured, but occure is what it says.

Reply:
I fixed that https://github.com/JDJGInc/JDBot/pull/9/commits/315b0782126ac36fe934ac3ba2d7132710d58651 here

Reply:
alrght, time to merge and test it.

Reply:
So i dunno how you didnt notice that

Reply:
Should be all good to @pieckenst so yeah :D